### PR TITLE
Add RFC for Docker files

### PR DESCRIPTION
**Pull request by jzabala**
Closes https://github.com/opencollective/opencollective/issues/2641

### CODE REVIEW

File: rfcs/007-docker-files.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- API
+
+## Motivation
+
+To make it as easy as possible to setup the api project locally.
+
+## Solution
+
+To leverage container technology to improve the API local setup.
```
[the remark this answers is not in the document]
Comment (jzabala) [reply rfcs/007-docker-files.md]: Solved.

File: rfcs/007-docker-files.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Suggestions to solve this limitation have come up; [#2531](https://github.com/opencollective/opencollective/issues/2531) and [#2556](https://github.com/opencollective/opencollective/issues/2556), but until we get there containers can help developers to have the environment they need by executing one command.
+
+- Easy to maintain
+
+This implementation provides predefined commands and the ability to run any current or future command using the same API as the predefined commands. The biggest point of change is the node version that needs to be updated in conjunction with the package.json and CircleCI config file.
+
+## Alternatives
+
+Some conversation have come up about creating [magic scripts](https://github.com/opencollective/opencollective/issues/2531#issuecomment-545900852) able to set up everything. (These would need to be OS dependent.)
```
[the remark this answers is not in the document]
Comment (jzabala) [reply rfcs/007-docker-files.md]: Thank you for pointing Vegrant. I didn't know that technology 🙂.

**Comment by jzabala** [comment]
Thank you very much for your comments @Betree

**Comment by jzabala** [comment]
Hey @Betree thank you for approving the PR 🙂.

What would be the next step? Do you want me to improve my propose implementation, add docs and submit the final PR to add the Docker files to the project?

**Comment by Betree** [comment]
Some docs update for the API would be welcome, but we should wait for @znarf's feedback before moving on with this one.

File: rfcs/007-docker-files.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# RFC 007 - Add Docker files to improve api setup
```
Comment (znarf) [inline rfcs/007-docker-files.md]:
```suggestion
# RFC 007 - Add Docker files to improve API setup
```

**Comment by jzabala** [comment]
> Some docs update for the API would be welcome, but we should wait for @znarf's feedback before moving on with this one.

I think it was my bad by starting a RFC without knowing the whole process. Sorry about that 😅

Since this RFC was approved I suppose you like the idea and want to go forward with it. So, how about if now I create two issues one in the API project and another in the documentation project, to officially introduce the change in the project?

Then you can merge and close this PR to introduce the RFC file, and I'll make reference to the RFC issue in the other issues I'll be creating.

**Comment by Betree** [comment]
> So, how about if now I create two issues: one in the API project and another in the documentation project, to officially introduce the change in the project?

Good idea.

I'm up to merge it right now so we can start experimenting.

**Pull request merged**